src/workbench-jupyter-docker/docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "2.4"
+services:
+  app:
+    container_name: "application-server"
+    image: "us-central1-docker.pkg.dev/verily-workbench-public/apps/workbench-jupyter:latest"
+    user: "jupyter"
+    restart: always
+    volumes:
+      - .:/workspace:cached
+      # Mount docker socket so application can access the docker daemon in the host
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Here is the snipped added to for docker to work. We still must install `docker` in the app to access the mounted daemon though.